src/components/NavBar.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
+function NavBar({navLinks, style}) {
+
+    const linksToDisplay = Object.entries(navLinks).map(([href, label]) => {
+        return (
+            <li key={href}>
+                <a href={href}>{label}</a>
+            </li>
+        )
+    })
```

Comment:
This is part 2 of the comment.

```suggestion
    const linksToDisplay = navLinks.forEach((link) => {
        return (
            <li key={link}>
                <a href="#{link.toLowerCase()}">{link}</a>
            </li>
        )
    })
```